pkg/roachprod/vm/flagstub/flagstub.go

```diff
@@ -28,6 +29,11 @@ type provider struct {
 	unimplemented string
 }
 
+// IsLocalProvider returns true because it is a local provider.
```

Comment:
This seems like a circular definition since it's a stub, neither local nor remote.